driver/conv_driver.hpp

```diff
@@ -125,6 +125,26 @@ struct AutoMiopenWarmupMode
     bool debug_find_enforce_disable_prev;
 };
 
+struct AutoConvDirectNaiveAlwaysEnable
+{
+    AutoConvDirectNaiveAlwaysEnable()
+    {
+        debug_always_enable_conv_direct_naive_prev = miopen::debug::AlwaysEnableConvDirectNaive;
+        miopen::debug::AlwaysEnableConvDirectNaive = true;
+    }
+    AutoConvDirectNaiveAlwaysEnable(const AutoConvDirectNaiveAlwaysEnable&) = delete;
+    AutoConvDirectNaiveAlwaysEnable(AutoConvDirectNaiveAlwaysEnable&&)      = delete;
+    AutoConvDirectNaiveAlwaysEnable& operator=(const AutoConvDirectNaiveAlwaysEnable&) = delete;
+    AutoConvDirectNaiveAlwaysEnable& operator=(AutoConvDirectNaiveAlwaysEnable&&) = delete;
+    ~AutoConvDirectNaiveAlwaysEnable()
+    {
+        miopen::debug::AlwaysEnableConvDirectNaive = debug_always_enable_conv_direct_naive_prev;
+    }
+
+    private:
+    bool debug_always_enable_conv_direct_naive_prev;
```

Comment:
[Recommendation] Use short names for small scopes. `prev` is enough. Or delete this. We simply need to swtch this debug flag ON and OFF. Let's follow the KISS principle.